src/main/java/com/marklogic/kafka/connect/source/MarkLogicSourceConfig.java

```diff
@@ -25,24 +25,26 @@ public class MarkLogicSourceConfig extends MarkLogicConfig {
     public static final ConfigDef CONFIG_DEF = getConfigDef();
 
     private static ConfigDef getConfigDef() {
-        ConfigDef configDef = new ConfigDef()
+        ConfigDef configDef = new ConfigDef();
+        MarkLogicConfig.addDefinitions(configDef);
+        return configDef
             .define(DSL_PLAN, Type.STRING, ConfigDef.NO_DEFAULT_VALUE, ConfigDef.CompositeValidator.of(new ConfigDef.NonNullValidator(), new ConfigDef.NonEmptyString()), Importance.HIGH,
-                "Required; a MarkLogic Optic DSL plan for querying the database")
-            .define(JOB_NAME, Type.STRING, "", new ConfigDef.NonNullValidator(), Importance.MEDIUM,
-                "Specifies the name for the query job executed by the connector task")
-            .define(CONSISTENT_SNAPSHOT, Type.BOOLEAN, true, new BooleanValidator(), Importance.MEDIUM,
-                "Controls whether to get an immutable view of the result set")
+                "Required; the Optic DSL query to execute")
             .define(TOPIC, Type.STRING, ConfigDef.NO_DEFAULT_VALUE, ConfigDef.CompositeValidator.of(new ConfigDef.NonNullValidator(), new ConfigDef.NonEmptyString()), Importance.HIGH,
-                "Required; the name of the target topic to publish records to")
+                "Required; the name of a Kafka topic to send records to")
```

Comment:
@BillFarber  I'm wondering if we should standardize on "send" or "publish". I don't know if Kafka is exclusively considered to be pub/sub so I opted for "send".

Reply:
Same Kafka web page: https://kafka.apache.org/08/documentation.html
I see both "send" and "publish" used almost interchangeably.

However, the Producer API:
public void send(KeyedMessage<K,V> message);

I do think we should standardize, but not sure on what.